pkg/sdk/tasks_gen_integration_test.go

```diff
@@ -0,0 +1,509 @@
+package sdk
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_Tasks(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	database, databaseCleanup := createDatabase(t, client)
+	t.Cleanup(databaseCleanup)
+
+	schema, schemaCleanup := createSchema(t, client, database)
+	t.Cleanup(schemaCleanup)
+
+	sql := "SELECT CURRENT_TIMESTAMP"
+
+	assertTask := func(t *testing.T, task *Task, id SchemaObjectIdentifier, name string) {
```

Comment:
there are so many of these functions... are these really so useful as opposed to declaring them in-line? and wouldn't it be more useful to have a function that could assert a createTaskRequest equals Task? Or for Alter, we probably don't need to check most attributes anyways, only the ones which have changed.

Reply:
Yeah, they are reused, that's why I have created them. I think we could have better, more generic assert, I will add it to our bag of SDK enhancements.

Reply:
Added as SNOW-933470.